tests/test-data/test_configs/consulting_blocklist.toml

```diff
@@ -0,0 +1,62 @@
+## Default zones, these should be present on all nameservers, except in rare
+##  configuration cases
+[[zones]]
+zone = "localhost"
+zone_type = "Primary"
+file = "default/localhost.zone"
+
+[[zones]]
+zone = "0.0.127.in-addr.arpa"
+zone_type = "Primary"
+file = "default/127.0.0.1.zone"
+
+[[zones]]
+zone = "0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa"
+zone_type = "Primary"
+file = "default/ipv6_1.zone"
+
+[[zones]]
+zone = "255.in-addr.arpa"
+zone_type = "Primary"
+file = "default/255.zone"
+
+[[zones]]
+zone = "0.in-addr.arpa"
+zone_type = "Primary"
+file = "default/0.zone"
+
+[[zones]]
+## zone: this is the ORIGIN of the zone, aka the base name, '.' is implied on the end
+##  specifying something other than '.' here, will restrict this recursor to only queries
+##  where the search name is a subzone of the name, e.g. if zone is "example.com.", then
+##  queries for "www.example.com" or "example.com" would be recursively queried.
+zone = "."
+
+## zone_type: Primary, Secondary, Hint, Forward
+zone_type = "Hint"
+
+[[zones.stores]]
+type = "recursor"
+roots = "default/root.zone"
+ns_cache_size = 1024
+record_cache_size = 1048576
+
+## do_not_query: these networks will not be sent queries during recursive resolution
+do_not_query = ["0.0.0.0/8", "127.0.0.0/8", "::/128", "::1/128"]
+
+## Note that in this example, the recursor comes first and the blocklist is second.  Queries
```

Comment:
Right now, it doesn't. I'll need to fix that.

Reply:
I don't think you need to "fix" it right now, I just was thinking about the advantage of this ordering as opposed to it before the recursor. It seems like if it's after the Recursor, this would be a nice feature. Your choice on making it work that way.